.formatter.exs

```diff
@@ -6,7 +6,10 @@
   ],
   inputs: [
     "*.{heex,ex,exs}",
-    "{config,apps,priv}/**/*.{heex,ex,exs}"
```

Comment:
I'm not sure what did it change? I think the old config would already guarantee that all hexx, ex and exs are formatted no matter where they were..

Reply:
Yeah it was pulling in matches from `deps/` and `node_modules/` folders that happened to match. Not sure why that started happening recently, but this fixed it. Didn't seem to be a clean way to omit certain directories.

Reply:
```
jamil@mbair:~/Developer/firezone/firezone (feat/api-token-cli$*) % find . -name "*.exs" | grep node_modules
./apps/fz_http/assets/node_modules/phoenix/mix.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.html/controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/confirmation_controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/forgot_password_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/confirmation_instructions_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/login_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/reset_password_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/registration_controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/conn_case.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/test_cases.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/reset_password_controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/auth_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/registration_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/confirmation_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/settings_controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/settings_live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.auth/session_controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.channel/channel_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.notifier/notifier_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.live/live_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.context/test_cases.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.context/context_test.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.schema/migration.exs
./apps/fz_http/assets/node_modules/phoenix/priv/templates/phx.gen.json/controller_test.exs
./apps/fz_http/assets/node_modules/phoenix/.formatter.exs
./apps/fz_http/assets/node_modules/phoenix_html/mix.exs
./apps/fz_http/assets/node_modules/phoenix_live_view/mix.exs
./apps/fz_http/assets/node_modules/phoenix_live_view/.formatter.exs
```